tests/resources/conformance_suites/dba/th/th06-testcase.xml

```diff
@@ -17,6 +17,8 @@
         </data>
         <result>
             <error>DBA.TH06</error>
+            <error>DBA.TR01</error>
+            <error>DBA.TR06</error>
```

Comment:
These are pretty simple validations to fix. Can we solve them so that the test case isnt polluted with other validations, please?

Reply:
Best I can do is fix DBA.TR01.  
DBA.TR06 compares the end date of gsd:IdentificationNumberCvrOfReportingEntity, which in this case is None as it has a forever context, to gsd:ReportingPeriodEndDate which can't be tagged as None without a schema error.

Reply:
That makes sense.